### PR TITLE
Update tools.sh to use consolidated

### PR DESCRIPTION
**Pull request by RSereno**
Update to use the correct file name `consolidated.*.pth`. tools.sh is still looking for `ggml-model-f16.bin*` "  skipping the file convertions. I wasn´t able to test this change  because for some reason when building the docker image in windows it can find the shell.

### CODE REVIEW

File: .devops/tools.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ elif [[ $arg1 == '--all-in-one' || $arg1 == '-a' ]]; then
     echo "Downloading model..."
     python3 ./download-pth.py "$1" "$2"
     echo "Converting PTH to GGML..."
-    for i in `ls $1/$2/ggml-model-f16.bin*`; do
+    for i in `ls $1/$2/consolidated.*.pth`; do
         if [ -f "${i/f16/q4_0}" ]; then
             echo "Skip model quantization, it already exists: ${i/f16/q4_0}"
         else
```
[the remark this answers is not in the document]
Comment (RSereno) [reply .devops/tools.sh:28]: You are right sw. I don't know much about the quantization yet to understand the name change.
Will consolidated.00.pth become ggml-model-q4_0 with the quatization?

[the remark this answers is not in the document]
Comment (RSereno) [reply .devops/tools.sh:28]: That's a good point. Will try to test it once I arrive home.

**Comment by RSereno** [comment]
I agree we should make it more clear, in the example:
"docker run -v /llama/models:/models ghcr.io/ggerganov/llama.cpp:full --all-in-one "/models/" 7B"
the models should be put in the folder:  /llama/models and it can be updated to where the models is saved.

The commandlines are indeed broken, i noticed before. I dont know how to resolve that one but I think that should be covered in a different PR. 
From the testing I did the string seems to break for example: "Building a website can be done in 10 simple steps:" the main will only get the "Building" has an argument.

**Comment by greigs** [comment]
This appears to have the wrong type of line endings in .devops/tools.sh - gives me the following error: "standard_init_linux.go:228: exec user process caused: no such file or directory"

This might be an OS specific error. I am running docker on windows with linux guest containers
I'm creating my own fork with the fix (tested)

**Comment by abasu0713** [comment]
+1 please. still getting the error. Thank you

**Comment by lmaddox** [comment]
> `-v /llama/models`

From where can I download the models, in order to populate that directory? Thanks.

**Pull request closed without merging**